ingestion/src/metadata/profiler/processor/sampler/sqlalchemy/snowflake/sampler.py

```diff
@@ -30,6 +33,11 @@ class SnowflakeSampler(SQASampler):
     """
 
     def __init__(self, *args, **kwargs) -> None:
+        self.sampling_method_type = SamplingMethodType.BERNOULLI
+
+        _profile_sample_config = kwargs.get("profile_sample_config", None)
+        if _profile_sample_config and _profile_sample_config.sampling_method_type:
+            self.sampling_method_type = _profile_sample_config.sampling_method_type
```

Comment:
I believe you can simplify this with 

```
self.sampling_method_type = kwargs.get("profile_sample_config", SamplingMethodType.BERNOULLI)
```

Reply:
I think that the value of `profile_sample_config` is assigned to the `self.sampling_method_type` if `profile_sample_config` is exist in the `kwargs`. But I want to assign the value of `profile_sample_config. sampling_method_type`.

Reply:
I tried your idea. 

```python
class SnowflakeSampler(SQASampler):
    def __init__(self, *args, **kwargs) -> None:
        if profile_sample_config:
            self.sampling_method_type = (
                profile_sample_config.sampling_method_type
                or SamplingMethodType.BERNOULLI
            )
        super().__init__(*args, **kwargs)
```

Next, I executed the test code for this code, and the error below occurred.

```
    def __init__(self, *args, **kwargs) -> None:
>       if profile_sample_config:
E       NameError: name 'profile_sample_config' is not defined
```

I think it is impossible to use local variables defined in another class even though it was a superclass.

Reply:
On the other hand, we can define arguments specifically like `BigQuerySampler`. I wonder which is better.
https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/profiler/processor/sampler/sqlalchemy/bigquery/sampler.py

Reply:
I fixed to specify init arguments to `profile_sample_config` directly.
How about it?

Reply:
The code below includes a problem that no value is applied if `profile_sample_config` is None.

```python
if  profile_sample_config:
    self.sampling_method_type = profile_sample_config.sampling_method_type or SamplingMethodType.BERNOULLI
```

I want to apply `BERNOULLI` to `sampling_method_type` as a default value.
That is why I have arranged your proposed code like below.

```python
self.sampling_method_type = SamplingMethodType.BERNOULLI
if profile_sample_config:
    self.sampling_method_type = profile_sample_config.sampling_method_type
```

Reply:
Thanks for the change. We'll merge it once the tests passed